src/pages/workspaces/workspace/Dashboard/WorkspaceNotifications.test.ts

```diff
@@ -25,7 +26,15 @@ describe('WorkspaceNotifications', () => {
   };
 
   afterEach(() => {
-    authStore.reset();
+    asMockedFn(Ajax).mockImplementation(
```

Comment:
If you move this to `beforeEach`, can you then remove the copy down below at line 29?

Reply:
It worked! Done!